ArduCopter/RC_Channel.cpp

```diff
@@ -155,14 +155,23 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
             }
             break;
 
-        case AUX_FUNC::SIMPLE_MODE:
+        case AUX_FUNC::SIMPLE_MODE: {
             // low = simple mode off, middle or high position turns simple mode on
-            copter.set_simple_mode(ch_flag == HIGH || ch_flag == MIDDLE);
+            Copter::SimpleMode mode = Copter::SimpleMode::OFF;
```

Comment:
This is OK, but maybe just use a ternary operator?

Reply:
@peterbarker san.
I agree. It can also be realized with ternary operators.
Because there is a process specified in the switch statement in the processing of the options of ch7 and 8, I thought that this expression is easier to understand.